examples/gocean/eg5/Makefile

```diff
@@ -66,35 +62,44 @@ KERNELS = init_field_mod.o update_field_mod.o
 # The name of the executable
 NAME=profile_test
 
-.PHONY: default template simple_timing drhook dl_timer
+.PHONY: all template simple_timing drhook dl_timer
+
+run: compile
+	./profile_test.template
+
+transform: psy.f90
 
 compile: template
 all: template simple_timing drhook dl_timer lfric
 
-template: PROFILE_DIR?=../../../lib/profiling/template
-template: PROFILE_LINK=-ldummy
-template: ../../../lib/profiling/template/libdummy.a $(NAME).template
+template: WRAPPER_DIR?=../../../lib/profiling/template
+template: WRAPPER_NAME=dummy
+template: PROFILE_LINK=
+template: $(NAME).template
 
 # Build the template profiling library
 ../../../lib/profiling/template/libdummy.a:
 	${MAKE} -C ../../../lib/profiling/template
 
-simple_timing: PROFILE_DIR?=../../../lib/profiling/simple_timing
-simple_timing: PROFILE_LINK=-lsimple_timing
+simple_timing: WRAPPER_DIR=../../../lib/profiling/simple_timing
+simple_timing: WRAPPER_NAME=simple_timing
+simple_timing: PROFILE_LINK=
 simple_timing: $(NAME).simple_timing
 
 # Needs to additionally link with drhook library. As default
 # assume it is installed 'next' to PSyclone
 drhook: DRHOOK_DIR?=../../../../drhook
-drhook: PROFILE_DIR=../../../lib/profiling/drhook
-drhook: PROFILE_LINK=-ldrhook_psy -L $(DRHOOK_DIR)/build/lib -ldrhook
+drhook: WRAPPER_DIR=../../../lib/profiling/drhook
+drhook: WRAPPER_NAME=drhook_psy
+drhook: PROFILE_LINK=-L $(DRHOOK_DIR)/build/lib -ldrhook
 drhook: $(NAME).drhook
 
 # Needs to compile with openmp and link in apeg-dl_timer library. As
 # default assume apeg-dl_timer is installed 'next' to PSyclone.
 dl_timer: DL_TIMER_ROOT?=../../../../apeg-dl_timer
```

Comment:
As with the other PR, the conditional assignment to DL_TIMER_ROOT doesn't seem to work for me.

Reply:
I believe this is part of the recent merged PR about auto-triggering on master. I'll merge master later, which should fix that.

Reply:
I just tested this again, and it seems to work now.